<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="1" column="0">
     <widget class="QPushButton" name="pb_clear">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QProgressBar" name="pBar_bar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="gb_input">
          <property name="title">
           <string>Input</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLabel" name="lb_inputText">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="le_inputText"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="lb_counter">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="spb_counter"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lb_log">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QTextEdit" name="te_log"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Отображение</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QListWidget" name="lw_list"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
       <widget class="QGroupBox" name="gb_flag">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>271</width>
          <height>321</height>
         </rect>
        </property>
        <property name="title">
         <string>Флаги</string>
        </property>
        <widget class="QCheckBox" name="cb_couter">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>90</y>
           <width>76</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Отк счетчик</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_bar">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>140</y>
           <width>76</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Отк бар</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="gb_rad">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>60</y>
          <width>411</width>
          <height>301</height>
         </rect>
        </property>
        <property name="title">
         <string>Радиокнопки</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
